app/src/main/java/grinnell/appdev/edu/lyles/Adapters/BeerAdapter.java

```diff
@@ -0,0 +1,82 @@
+package grinnell.appdev.edu.lyles.adapters;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.media.Image;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ImageButton;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import grinnell.appdev.edu.lyles.R;
+import grinnell.appdev.edu.lyles.menuItems.BeerItem;
+
+/**
+ * Created by mattmurphy on 9/11/16.
+ *
+ * General adapter to be used by beer
+ * adapter contains image, and description of catelog item
+ */
+
+public class BeerAdapter extends ArrayAdapter<BeerItem> {
```

Comment:
Nice but this pattern of adapter creation is outmoded. Rather than using an ArrayAdapter<CustomObject>, you should extend the `RecyclerView.Adapter<CustomObject> class.  More on this here: https://guides.codepath.com/android/Using-the-RecyclerView